TelCo.ColorCoder/ManualMap.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+  class ManualMap{
+    
+    public static void ToString(){
+      for(int pairNumber = 1; pairNumber < 26; pairNumber++){
```

Comment:
26? the check should be <=25. Know why?

These are called magic numbers ideally you should not hardcode these numbers ad define constants in the beginning.
So when you define a constant you would define as
const int minPairNumber = 1;
const int maxPairNumber = 25;
you wouldnt have oneMoreThanMaxPainrNumber = 26 right?
So the check even if you hardcode shouldbe pairNumber <= 25 for readability purpose